src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -33,4 +33,7 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    public String getFeedbackToUser() {
```

Comment:
Correct! You now need a getter method for outsider.

However, you did not update the users of the original feedbackToUser field.